portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,57 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+document.addEventListener("DOMContentLoaded", function() { 
+  let currSlide = 1;
+  let numImgs = 3;
+
+  createSlide(currSlide, numImgs);
+
+  const prev = document.getElementById("prev");
+  const next = document.getElementById("next");
+
+  function changeSlide (direction) {
+    currSlide += direction;
+    if(currSlide < 1) {
+      currSlide = numImgs;
+    }
+    if(currSlide > numImgs) {
+      currSlide = 1;
+    }
+    document.getElementById("slide-img").src = `images/img${currSlide}.jpg`;
+    document.getElementById("slide-number").innerHTML = `${currSlide},${numImgs}`;
+  }
+
+  prev.onclick = () => {
+    changeSlide(-1);
+  };
+
+  next.onclick = () => {
+    changeSlide(1);
+  };
+});
+
+function createSlide(currSlide, numImgs) {
+  let slide = document.getElementById("slide");
```

Comment:
use const here instead of let

you should use const if the variable won't be reassigned (but it can be mutated!!!). let means that you expect the variable to be reassigned.

use this elsewhere as well